examples/angular/src/app/AppConfigLocation/appConfigLocation.component.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Why the nested `<div>`?  Can't we keep it clean and simply render `<p>Hello Config (AppId {{ sdk.ids.app }})</p>` which roughly matches what our TypeScript example does